File: api-gateway/src/main/java/com/kjj/apigateway/jwt/JwtTemplate.java
package com.kjj.apigateway.jwt;

import org.springframework.stereotype.Component;

@Component
public class JwtTemplate {
    public String getSecret() { return "capston2023kjj";}
    public String getTokenPrefix() {
        return "Bearer ";
    }

    public String getClaimUsername() { return "username"; }
    public String getClaimRoles() { return "roles"; }
    public String getClaimId() { return "id"; }
}

File: gateway/src/main/kotlin/com/kjj/gateway/jwt/JwtTemplate.kt
package com.kjj.gateway.jwt

import org.springframework.stereotype.Component

@Component
data class JwtTemplate (
    val secret: String = "capston2023kjj",
    val tokenPrefix: String = "Bearer ",
    val claimUsername: String = "username",
    val claimExp: String = "exp",
    val claimRoles: String = "roles",
    val claimId: String = "id"
)